tools/pipelines/build-client.yml

```diff
@@ -200,7 +200,7 @@ extends:
     publishDocs: true
     # We only care about pipeline run telemetry for the CI runs on the internal project, not for PR runs in the public
     # one. And since we don't batch commits for CI pipeline runs, the only reason we see for automated runs is IndividualCI.
-    telemetry: ${{ eq(variables['Build.Reason'], 'IndividualCI') }}
+    telemetry: true # ${{ eq(variables['Build.Reason'], 'IndividualCI') }}
     taskTest:
```

Comment:
Stale debugging change?

Reply:
Not stale yet, I'm still trying to get a successful run of build - client so I can do a final set of test pipeline runs that validate the telemetry upload is working correctly everywhere I touched. But it's going away before merge, yes.